src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -34,8 +34,8 @@
       <ProtoRoot>Otlp</ProtoRoot>
     </Protobuf>
 
-    <Protobuf Include="..\Aspire.Hosting\Dashboard\proto\resource_service.proto">
-      <Link>ResourceService\resource_service.proto</Link>
+    <Protobuf Include="..\Aspire.Hosting\Dashboard\proto\dashboard_service.proto">
```

Comment:
I renamed this file. The file was called resource_service, but the service inside it is called `DashboardService`.

Reply:
Rename the service, this is a better name

Reply:
Renaming the service is a wire breaking change. Requests use the service name in generated URLs.